anitya/lib/ecosystems/crates.py

```diff
@@ -28,3 +28,4 @@ class CratesEcosystem(BaseEcosystem):
 
     name = 'crates.io'
     default_backend = 'crates.io'
+    librariesio_name = 'Cargo'
```

Comment:
Yeah I'm not pleased with this either, it was just the first thing that popped into my head. Having an "aliases" field definitely sounds better to me.